intents.go

```diff
@@ -14,8 +14,10 @@ import (
 type AssetType string
 
 const (
-	TokenType AssetType = "TOKEN"
-	StakeType AssetType = "STAKE"
+	TokenType          AssetType = "TOKEN"
+	StakeType          AssetType = "STAKE"
+	SupplyType         AssetType = "SUPPLY"
+	WithdrawSupplyType AssetType = "WITHDRAW_SUPPLY"
 )
```

Comment:
AssetType = "WITHDRAW"

Reply:
done in https://github.com/blndgs/model/pull/23/commits/a4180516f38df4f16244c106c6668df00c85fa10